### ISSUE TITLE
Some problems with the prediction results of Ref-NeRF in Shiny Blender Dataset

### ISSUE THREAD

**Issue by Riga27527** [google-research/multinerf#86]
I want to reproduce the prediction results of Ref-NeRF in the paper, but due to some limitations of the hardware, I need to adjust the batch size and learning rate during the training process. I set the batch size to 1/8 of the original, and the learning rate to 1/8 of the original, according to the principle of linear scaling. The number of iteration rounds is set to 500k. However, after training the toaster, coffee and teapot scenarios, we found that the prediction results still have a certain gap with the paper, especially the MAE of the predicted normal. The following are our experimental settings and results.

The experimental configuration we changed :
  batch_size: int = 2048 
  lr_init: float = 0.00025
  lr_final: float = 2.5e-06
  render_chunk_size: int = 2048
  max_steps: int = 500000
  NerfMLP.net_width_viewdirs = 256
  NerfMLP.bottleneck_width = 256
  Config.eval_render_interval = 1

Our experimental results :

![metrics](https://user-images.githubusercontent.com/55527886/209513232-ee012c57-ea84-45a7-970b-5ca19279eb7b.png)

We found that our MAE metric for predicting normals is quite different from that in the paper.

We want to confirm whether there is a major problem with such a configuration, or whether these experimental results are normal.

Thanks a lot !!!
